File: _정답코드/2주차/02_카운팅_정렬_2/solution.py
def solution(numbers):
    count = [0] * 2001

    for num in numbers:
        count[num+1000] += 1
    
    for idx in range(1, 2001):
        count[idx] += count[idx-1]
    
    answer = [0] * len(numbers)
    for num in numbers:
        answer[count[num+1000]-1] = num
        count[num+1000] -= 1

    return answer

File: _정답코드/2주차/02_카운팅_정렬_2/solution.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> numbers) {
    vector<int> count = vector<int>(2001, 0);
	vector<int> answer = vector<int>(numbers.size());

    for(int num : numbers){
        count[num+1000] += 1;
    }

    for(int idx = 1; idx < 2001; idx++){
        count[idx] += count[idx-1];
    }

    for(int num : numbers){
        answer[count[num+1000] - 1] = num;
        count[num+1000] -= 1;
    }
	return answer;
}

